docs/tutorial_files/summer_triangle--tutorial.rst

```diff
@@ -0,0 +1,354 @@
+    from astroplan.plots import plot_airmass
+    import matplotlib.pyplot as plt
+
+    plot_airmass(altair, subaru, time)
+    plot_airmass(vega, subaru, time)
+    plot_airmass(deneb, subaru, time)
+
+    # Note that you don't need the ax commands below to produce the plot.
+    # They reduce the plot size for the documentation.
+    ax = plt.gca()
+    box = ax.get_position()
+    ax.set_position([box.x0, box.y0, box.width * 0.8, box.height * 0.8])
+
+    plt.legend(loc=1, bbox_to_anchor=(1.35, 1))
+    plt.show()
+
+We want a minimum airmass when observing, and it looks like sometime between
+9:00 and 15:00 UTC (or 23:00 on the 15th to 5:00 on the 16th, US/Hawaii) would
+be the best time to observe all three targets.
+
+However, if we want to define a more specific time window based on airmass, we
+can calculate this quantity directly.
+
+To get airmass measurements, we have to go through the `altaz` frame:
+
+.. code-block:: ipython
+
+    In [172]: subaru.altaz(time, altair).secz
+    Out[172]: 1.030256
+
+    In [173]: subaru.altaz(time, vega).secz
+    Out[173]: 1.0690128
+
+    In [174]: subaru.altaz(time, deneb).secz
+    Out[174]: 1.1677464
+
+Parallactic Angle
+-----------------
+
+To get a general idea of our targets' parallactic angle on the night of
+observation, we can make a plot::
+
+    from astroplan.plots import plot_parallactic
+
+    plot_parallactic(altair, subaru, time)
+    plot_parallactic(vega, subaru, time)
+    plot_parallactic(deneb, subaru, time)
+
+    plt.legend(loc=2)
+    plt.show()
+
+.. plot::
+
+    import astropy.units as u
+    from astropy.coordinates import EarthLocation
+    from pytz import timezone
+    from astroplan import Observer
+
+    longitude = '-155d28m48.900s'
+    latitude = '+19d49m42.600s'
+    elevation = 4163 * u.m
+    location = EarthLocation.from_geodetic(longitude, latitude, elevation)
+
+    subaru = Observer(name='Subaru Telescope',
+                   location=location,
+                   timezone=timezone('US/Hawaii'),
+                   description="Subaru Telescope on Mauna Kea, Hawaii")
+
+    from astropy.coordinates import SkyCoord
+    from astroplan import FixedTarget
+
+    coordinates = SkyCoord('19h50m47.6s', '+08d52m12.0s', frame='icrs')
+    altair = FixedTarget(name='Altair', coord=coordinates)
+
+    coordinates = SkyCoord('18h36m56.5s', '+38d47m06.6s', frame='icrs')
+    vega = FixedTarget(name='Vega', coord=coordinates)
+
+    coordinates = SkyCoord('20h41m25.9s', '+45d16m49.3s', frame='icrs')
+    deneb = FixedTarget(name='Deneb', coord=coordinates)
+
+    from astropy.time import Time
+
+    time = Time('2015-06-16 12:00:00')
+
+    from astroplan.plots import plot_parallactic
+    import matplotlib.pyplot as plt
+
+    plot_parallactic(altair, subaru, time)
+    plot_parallactic(vega, subaru, time)
+    plot_parallactic(deneb, subaru, time)
+
+    plt.legend(loc=2)
+    plt.show()
+
+We can also calculate this quantity directly:
+
+.. code-block:: ipython
```

Comment:
I haven't thought about it much, but I think I'm -1 on using the `ipython` Sphinx directive.

As far as I can see, the main difference wrt. the normal Python Sphinx directive is the input / output cell numbers.
And those don't carry useful info, i.e. are more distracting than helpful in my opinion.

So I'd suggest re-writing the examples that show input and output using the Python prompt `>>>` prefix for inputs:

```
.. code-block:: python
    >>> subaru.parallactic_angle(time, altair)
    −0.640582rad
    >>> subaru.parallactic_angle(time, vega)
    −0.465298rad
```
If it's about getting this format quickly for the docs, what I sometimes do is call `%history` in IPython, and then copy & paste the relevant commands into a new Python shell, then copy & paste & indent by 4 spaces that to the RST file.

If you agree to this change, the point is irrelevant. But in case not, the IPython directive is currently not found in the Sphinx build, resulting in this error:
https://travis-ci.org/astroplanners/astroplan/jobs/74846421#L874